src/pal/src/loader/module.cpp

```diff
@@ -1620,6 +1620,8 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
         shortAsciiName = FREEBSD_LIBC;
 #elif defined(__NetBSD__)
         shortAsciiName = "libc.so";
+#elif !HAVE_GNU_LIBNAMES_H
+        shortAsciiName = "libc.musl-x86_64.so";
```

Comment:
@rofl0r, the module loader requires CRT lib on the target platform. On Alpine libc.musl-x86_64.so.1 is the one which ldd is showing for every compiled binary, hence i used it.